services/web/server/src/simcore_service_webserver/catalog.py

```diff
@@ -159,23 +90,34 @@ def setup_catalog(app: web.Application, *, disable_auth=False):
     # ---------------------------------------------
 
     # resolve url
-    app[f"{__name__}.catalog_origin"] = URL.build(
-        scheme="http", host=cfg["host"], port=cfg["port"]
-    )
-    app[f"{__name__}.catalog_version_prefix"] = cfg["version"]
+    app[KCATALOG_ORIGIN] = URL.build(scheme="http", host=cfg["host"], port=cfg["port"])
+    app[KCATALOG_VERSION_PREFIX] = cfg["version"]
 
     specs = app[APP_OPENAPI_SPECS_KEY]  # validated openapi specs
 
-    # bind routes with handlers
+    # TODO: disable when feature is released
+    exclude: List[str] = []
+    if strtobool(os.environ.get("WEBSERVER_DEV_FEATURES_ENABLED", "0")):
```

Comment:
Yes, I actually did it that way at first reverted to this because:
1. app settings in webserver are still based in trafaret (did only first iteration)
2. this is just going to be there for a couple of PR until the feature in the front-end is available

so the RoI was too low and exceptionally decided to go this way